src/App.tsx

```diff
@@ -1,24 +1,238 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import { useState, useEffect } from 'react';
+import { Header } from './components/Header/Header';
+import { Footer } from './components/Footer/Footer';
+import { TodoList } from './components/TodoList/TodoList';
+import { ErrorNotification }
+  from './components/ErrorNotification/ErrorNotification';
 
-const USER_ID = 0;
+import {
+  getTodos, addTodo, deleteTodo, updateStatusTodo, updateTitleTodo,
+} from './api/todos';
+
+import { Todo } from './types/Todo';
+import { SortType } from './types/SortType';
+
+const USER_ID = 6650;
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
-    return <UserWarning />;
+  const [todosList, setTodosList] = useState<Todo[]>([]);
+  const [query, setQuery] = useState('');
+  const [isError, setIsError] = useState(false);
+  const [errorType, setErrorType] = useState('');
```

Comment:
just check does errorType contain any value, instead of isError state

Reply:
fixed